tmp/build/patch-charts.sh

```diff
@@ -78,37 +66,17 @@ function patchTemplates() {
                   s/\(\(^ *\)- port: 443\)/\1\
 \2  targetPort: 8443/
                 }
-              }' ${HELM_DIR}/istio/charts/gateways/values.yaml
+             }' ${HELM_DIR}/istio/charts/gateways/values.yaml
+
   sed -i -e 's/\(^ *\)- containerPort: {{ $val.port }}/\1- name: {{ $val.name }}\
 \1  containerPort: {{ $val.targetPort | default $val.port }}/' ${HELM_DIR}/istio/charts/gateways/templates/deployment.yaml
-  # add support for IOR
-  sed -i -e '/istio-ingressgateway:/,/enabled:/ {
-    /enabled:/ a\
-\  # set to true to enable route creation\
-\  ior_enabled: false\
-\  ior_image: istio-ior-rhel8\
-
-  }' ${HELM_DIR}/istio/charts/gateways/values.yaml
-  if [[ "${COMMUNITY,,}" == "true" ]]; then
-    sed -i -e 's/ior_image:.*$/ior_image: istio-ior-ubi8/' ${HELM_DIR}/istio/charts/gateways/values.yaml
-  fi
 
-  # enable ingress for tracing
-  sed -i -e '/ingress:/,/enabled/ { s/enabled: .*$/enabled: true/ }' ${HELM_DIR}/istio/charts/tracing/values.yaml
-
-  # enable ingress for kaili
-  # update hub/tag
-  sed -i -e '/ingress:/,/enabled/ { s/enabled: .*$/enabled: true/ }' ${HELM_DIR}/istio/charts/kiali/values.yaml
-  if [[ "${COMMUNITY,,}" == "true" ]]; then
-    sed -i -e 's/hub:.*$/hub: kiali/' \
-           -e 's/tag:.*$/tag: v'${KIALI_VERSION}'/' ${HELM_DIR}/istio/charts/kiali/values.yaml
-  else
-    sed -i -e 's+hub:.*$+hub: registry\.redhat\.io\/openshift-service-mesh+' \
-           -e 's/tag:.*$/tag: '${KIALI_VERSION}'/' ${HELM_DIR}/istio/charts/kiali/values.yaml
-  fi
-
-  # In Istio 1.2, this viewOnlyMode is there, but in Istio 1.1 we need to add it - it is supported by the latest Kiali
-  sed -i -e 's/grafanaURL:/viewOnlyMode: false\n  grafanaURL:/' ${HELM_DIR}/istio/charts/kiali/values.yaml
+	#CNI is handled separately
```

Comment:
I don't think this is needed anymore -- cni image location is now being specified in the operator deployment ENV.

Reply:
Yeah, I'm not sure so I added it back in to be safe. That said, I haven't been able to find anywhere it's actually used.